src/libs/Network/NetworkStore.ts

```diff
@@ -82,85 +78,51 @@ Onyx.connect({
             triggerReconnectCallback();
         }
 
-        offline = Boolean(network.shouldForceOffline) || network.isOffline;
+        offline = Boolean(network.shouldForceOffline) || !!network.isOffline;
```

Comment:
Why this `!!`? Isn't `isOffline` a boolean?

Reply:
`isOffline` is optional, similar [explanation](https://github.com/Expensify/App/pull/28031#discussion_r1345573497). If you are certain it is required we can change it.